src/services/fileStorage/hooks/index.js

```diff
@@ -34,9 +41,9 @@ exports.before = {
 	],
 	find: [hasPermission('FILESTORAGE_VIEW')],
 	get: [hasPermission('FILESTORAGE_VIEW')],
-	create: [hasPermission('FILESTORAGE_CREATE')],
+	create: [hasPermission('FILESTORAGE_CREATE'), replaceAnd],
 	update: [hasPermission('FILESTORAGE_EDIT')],
```

Comment:
`replaceAnd` should also be used on update, shouldn't it?

Reply:
sure, haven't seen anyone using update yet, but you're right.